src/bindings/python/tests/test_direct_comparison/test_partial_shape.py

```diff
@@ -0,0 +1,15 @@
+import pytest
```

Comment:
Please, move these tests to [this file](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_graph/test_core.py#L112).

Reply:
Done! Thanks for the suggestion.